generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -6,6 +6,10 @@ import { TranslateService } from 'ng2-translate';
 <%_ } _%>
 import { AlertService } from 'ng-jhipster';
 
+<%_ if (websocket === 'spring-websocket') { _%>
+import { WindowRef } from './tracker/window.service';
```

Comment:
Ok seems its already removed so its only used here. But i think this could still be moved to the lib. what do you think?